### DISCUSSION TITLE
Problem in Lava STDP tutorial

### DISCUSSION THREAD

**Discussion by Rashedape** [lava-nc/lava#727] [General]
Hello,
I am currently practicing STDP tutorial [https://lava-nc.org/lava/notebooks/in_depth/tutorial08_stdp.html](url) given in the lava site. But i am getting a weight changes graph which is not similar. 
![lava](https://github.com/lava-nc/lava/assets/95880434/4eb32261-a56d-4c69-83fc-e65d3471bb95)
How can i get the same weight changes graph? Can anyone give me any suggestion?

**Comment by weidel-p** [comment]
Hi,

hard to be sure from just seeing a figure, but it looks like that there is a sign error somewhere. Maybe your learning rule is supposed to look like this:

```
dw = "-2 * x0 * y1 + 2 * y0 * x1"
```

but looks like this:

```
dw = "2 * x0 * y1 - 2 * y0 * x1"

```

**Reply by Rashedape** [reply]
I did not change the code given on the Lava site. I tried to create a new environment, but nothing happens.

**Reply by PhilippPlank** [reply]
![image](https://github.com/lava-nc/lava/assets/32519998/fcda9be6-5ebe-4975-89fc-2bec12bad9b0)

If you change the sign of A_plus and A_minus in this cell of the notebook, what is the result? Could you also tell us which Lava version and which system/OS you are using?
Thanks!

**Reply by Rashedape** [reply]
I changed the sign but it did not provide the corrected graph. 
Lava version 0.5.0
OS- Ubuntu
Thanks

**Reply by PhilippPlank** [reply]
Ah, could you please upgrade to Lava 0.7.0? There was a problem in the older version, using the newer release or the latest version from GitHub should fix this issue.
Thanks!

**Reply by Rashedape** [reply]
The problem is solved. Thank You.
